data/IntroText.h

```diff
@@ -21,7 +21,7 @@ const char *const introTextData =
 	"\n"
 	"Contributors: \bgStanislaw K Skowronek (Designed the original Powder Toy),\n"
 	"\bgSimon Robertshaw, Skresanov Savely, cracker64, Catelite, Bryan Hoyle, Nathan Cousins, jacksonmj,\n"
-	"\bgFelix Wallin, Lieuwe Mosch, Anthony Boot, Matthew \"me4502\", MaksProg, jacob1, mniip, LBPHacker\n"
+	"\bgFelix Wallin, Lieuwe Mosch, Anthony Boot, Matthew \"me4502\", MaksProg, jacob1, mniip, LBPHacker, RCAProduction\n"
```

Comment:
We'll let you know when it's appropriate to add your name to the contributors list. One element is not enough though. There are a huge amount of other people who have submitted elements but don't have their name in this list yet (like moony).

Reply:
I have been summoned.
(And now I shall leave)